docs/components/modeler/bpmn/business-rule-tasks/business-rule-tasks.md

```diff
@@ -35,8 +85,6 @@ Define [variable mappings](/components/concepts/variables.md#inputoutput-variabl
 the [same way as a service task does](/components/modeler/bpmn/service-tasks/service-tasks.md#variable-mappings)
 to transform the variables passed to the job worker, or to customize how the variables of the job merge.
 
-## Additional resources
-
 :::tip Community Extension
```

Comment:
:wrench: We could remove the note that highlights the community extension. With the internal DMN engine, it is unlikely that a user wants to use the extension.

Reply:
🤔 I was leaving it in because it offered an example for implementing a job worker, but I guess we should provide such examples in the service task documentation

Reply:
:+1: